src/Annotator.js

```diff
@@ -212,7 +215,6 @@ class Annotator extends EventEmitter {
      */
     setupControllers() {
         this.modeButtons = this.options.modeButtons || {};
-
```

Comment:
We generally try to keep whitespace between unrelated blocks, like property accessors, variable declarations, returns, etc.